wandb/sdk/launch/runner/kubernetes.py

```diff
@@ -239,7 +233,8 @@ def run(self, launch_project: LaunchProject) -> Optional[AbstractRun]:  # noqa:
             if registry:
                 repo, tag = image_uri.split(":")
                 docker.push(repo, tag)
-
+        env_vars = get_env_vars_dict(launch_project, self._api)
+        containers[0]["env"] = [{"name": k, "value": v} for k, v in env_vars.items()]
```

Comment:
we should also have a line here allowing users to specify additional env vars

Reply:
Sounds good